src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelBindingResult.cs

```diff
@@ -130,6 +116,23 @@ private ModelBindingResult(string key, object model, bool isModelSet, ModelValid
         public ModelValidationNode ValidationNode { get; }
 
         /// <inheritdoc />
+        public override string ToString()
```

Comment:
Yes, and also without this the messages printed by xUnit when you have a test failure are pretty un-actionable.